lib/commands/element.js

```diff
@@ -0,0 +1,22 @@
+import { errors } from 'mobile-json-wire-protocol';
+import _ from 'lodash';
+
+let commands = {}, helpers = {}, extensions = {};
+
+commands.getAttribute = async function (attribute, elementId) {
+  if (this.curContext) {
+    /* TODO */throw new errors.NotYetImplementedError();
+    // return await this.findWebElementOrElements(strategy, selector, mult, context);
+  } else {
+    if (_.contains(['label', 'name', 'value', 'values', 'hint'], attribute)) {
+      let command = ["au.getElement('", elementId, "').", attribute, "()"].join('');
```

Comment:
This sort of thing that we did in the old Appium would be much easier to read using string interpolation.